dist/immstruct.js

```diff
@@ -653,13 +824,12 @@ function Structure (options) {
     this._currentRevision = 0;
   }
 
-  this._pathListeners = [];
+  this._pathListeners = {};
```

Comment:
oops, thats actually inaccurate, it works top down not bottom up, I was doing a reverse on the resulting array of callbacks but to simulate a bottom up but it did not like that.